### ISSUE TITLE
Strip triples automatically added to rdf type

### ISSUE THREAD

**Issue by lsat12357** [OregonDigital/oregondigital_2#48]

**Comment by mickeroo** [comment]
@terrellt is this something we still need to do?

**Comment by terrellt** [comment]
Yes. I'd like there to be functionality in AF to handle this, but it's a struggle.

**Comment by srabun** [comment]
There are places in public view where you see internal fedora data - rdf type? and other stuff...might have already been cleaned up in AF, but...
